### ISSUE TITLE
Please add the ability to edit relation meta field (relation meta data) in forms and the ability to disconnect relations in forms.

### ISSUE THREAD

**Issue by cmw1990** [CrocoBlock/suggestions#5513]
.

**Comment by Semonxue** [comment]
Hi @cmw1990 , I agree with you for the first suggestion. I also need edit and filter relation meta in front page. But disconnect the relation is available in jetformbuilder.

**Comment by lunikfrank** [comment]
I agree, having the possibility to edit relation meta data in JetFormsBuilder would be awesome (Since Forms is now legacy). I can see the complexity of doing so when there are multiple related items, but at least if it would be possible to edit 1 related item at a time. 

**Here's a use case example for which i'd like to use it.**
On my website, I want to connect parent events (CPT) with children Artists (Users). The relation between the two would have a true/false meta field wether the artist confirms his presence to the event and i'd like for the artist, on the front end, to simply be able to set the true/false meta of the relationship. 

This is one example, but I can see so many other opportunities with such a feature.

I could see as a possibility maybe in jetEngine queries for related items returned data to add "Current Parent ID" and then, for example, when using a form tied to a listing grid item, to make it so that both the Parent ID and Child ID are available to the form as "hidden" relation data. Then, in the "Connect relation items" post action, add the possibility to set meta-field value either manually or based on a form field?

![image](https://github.com/Crocoblock/suggestions/assets/33937638/cbb82507-6684-46a7-af8f-95d3ed870e0a)

**Comment by MechelAbdo** [comment]
+1

**Comment by hvrddrive** [comment]
@Crocoblock @MjHead Any update on this? This would be very useful to be able to update the meta fields for relations in Jet Form Builder.

**Comment by joelgaribaygz** [comment]
Any news on this request? I've been requesting this via Support for a long time and I see that it's already here since 2022...

**Comment by dominikgraef** [comment]
This feature would be very awesome and would breaking some roadblocks in the conception and development process, especially in the topics about user-side management possibilities at the frontend (and no backend access is necesary)

**Comment by joesandin** [comment]
I agree this is a very needed feature and makes it hard to work with relationship meta fields on the front end. In my case, I have a relationship between users and companies and there is a meta field that defines the relationship, such as "Account Owner", "Client Success Manager", etc. If I want to allow someone on the front end to add a relationship and specify one of these roles, I can't find a way to do so. They can only add the relationship with no role value. Has anyone found a workaround to this?

**Comment by Aldermoon2** [comment]
It has been added quite some ago to JetFormBuilder, as requested. https://crocoblock.com/knowledge-base/jetengine/how-to-disconnect-related-items-from-the-front-end-via-form/

**Comment by BoostMyCroco** [comment]
Hi !!

Here are two ways to do it:

One mentioned by Victoria Bormotova in this post 

[Post FB Group](https://web.facebook.com/share/p/18au5PCURD/) 

And another one to do it with queries and a Hook instead of API:

[Video BoostMyCroco YT Channel](https://youtu.be/K0WproXCh24)

**Comment by BoostMyCroco** [comment]
Hi Guys,

This method may be useful while CB releases an official fix:

[https://youtu.be/K0WproXCh24](https://youtu.be/K0WproXCh24)

**Comment by tonnysantana2004** [comment]
@BoostMyCroco God bless you!

**Comment by NaorWbd** [comment]
> Hi Guys,
> 
> This method may be useful while CB releases an official fix:
> 
> https://youtu.be/K0WproXCh24

Thanks for the assistance! Unfortunately, this youtube channel has closed. Still need to figure out how to solve it
